cli/atlas/templates/Makefile.gotmpl

```diff
@@ -1,76 +1,53 @@
-PROJECT_ROOT 				:= {{ if .Root }}{{ .Root }}/{{ .Name }}{{ else }}{{ .Name }}{{ end }}
-BUILD_PATH  				:= bin
-DOCKERFILE_PATH 		:= $(CURDIR)/docker
+PROJECT_ROOT		:= {{ if .Root }}{{ .Root }}/{{ .Name }}{{ else }}{{ .Name }}{{ end }}
+BUILD_PATH		:= bin
+DOCKERFILE_PATH		:= $(CURDIR)/docker
```

Comment:
Why is the spacing/tab different here?

Reply:
I think the git diff is causing indentation issues, but I'm not sure why. The [template itself](https://github.com/ZachEddy/atlas-app-toolkit/blob/1e1677af0e40fcc28b2288b3fb2dc37e4b255326/cli/atlas/templates/Makefile.gotmpl) doesn't have spacing/tab problems and it renders [Makefiles](https://github.com/ZachEddy/atlas-app-toolkit/blob/remove-buildtool-example/cli/example/Makefile) that look okay.